l10n_jp_address_layout/__manifest__.py

```diff
@@ -1,23 +1,16 @@
-# -*- coding: utf-8 -*-
```

Comment:
This module is an OCA module, i.e. https://github.com/OCA/l10n-japan/tree/11.0/l10n_jp_address_layout
Please exclude it from pre-commit and revert to the original version.